openmc/mgxs/mgxs.py

```diff
@@ -548,26 +548,14 @@ def get_nuclide_density(self, nuclide):
         float
             The atomic number density (atom/b-cm) for the nuclide of interest
 
-        Raises
```

Comment:
Looks like this would require changes in `Tally.get_values`. I'm going to defer for now until someone complains about it :)